doc/development_process.md

```diff
@@ -0,0 +1,165 @@
+
+On GitHub it can be a little harder to tell who is who, and we don't really have a way around that.  If you see somebody merging PRs, closing issue posts, or adding labels to issues and PRs though, that's a person who probably has a higher level of trust in the project.  There's also a "member" badge on the upper right of github posts that indicates a person is a member of Cleverraven with at least a little authority, like this:
+![image](https://github.com/I-am-Erk/Cataclysm-DDA/assets/45136638/6106b80f-274e-43c7-a937-58f601f035e6)  
+If a person has a "member" badge there, it usually means they speak with some authority.  However, we all have different areas of the project we know better than others, so if a member of cleverraven says something like "I don't know much about armpit code, but maybe you want to do X", they're trying to tell you not to take their word too seriously.
+
+#### My idea was rejected, changed beyond recognition, or requires a lot more than I expected.
+
+This can happen for a few reasons.  If you're not sure which applies to you, feel free to ask for clarification.
+
+* Sometimes, your idea sparks a huge conversation, and the next thing you know it seems like you're on the hook to overhaul armpit fart mechanics before you can start.  *Please, feel free to mention if you feel this way*.  This usually means you've hit something cool that we're all excited about.  It often **doesn't mean we expect you to add** whatever we've started riffing on, and you're fine to just put in whatever small thing you originally wanted.  We're all just huge nerds, and we know that sometimes we get carried away.
+* Sometimes, what you want to add seems simple, but the simple solution has already been rejected because of complex flaws.  Often it's better not to add a simplified version that will cause us problems down the road, because we then need to either revert or rework that solution to go on.  If you're not able to do the harder work, you'll have to find a different idea.  We won't think less of you for that, and we're sorry when this happens.  It usually means we want the thing you want too.
+* Sometimes, what you want just isn't compatible with the core game.  It might go fine in a mod, and it might even [be fine to include that mod in the main repository](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/IN_REPO_MODS.md) depending on what it is.  Remember that almost all the senior devs have our own pet mods; sometimes, things just go better there, and that's okay.
```

Comment:
```suggestion
* Sometimes, what you want just isn't compatible with the core game.  It might go fine in a mod, and it might even be fine to include that mod in the main repository (https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/IN_REPO_MODS.md) depending on what it is.  Remember that almost all the senior devs have our own pet mods; sometimes, things just go better there, and that's okay.
```